File: client/src/main/java/com/github/Diosa34/DMS_HelloWorld/classes/Coordinates.java
package com.github.Diosa34.DMS_HelloWorld.classes;

/**
 * Entity coordinate class
 */
public class Coordinates {
    private final Float x; //Поле не может быть null
    private final Integer y; //Поле не может быть null

    public Coordinates(Float x, Integer y){
        this.x = x;
        this.y = y;
    }

    public String toString() {
        return this.x + ", " + this.y;
    }
}

File: common/src/main/kotlin/com/github/Diosa34/DMS_HelloWorld/collection/Coordinates.kt
package com.github.Diosa34.DMS_HelloWorld.collection

import io.github.landgrafhomyak.itmo.dms_lab.interop.DisplayName
import kotlinx.serialization.Serializable

/**
 * Entity coordinate class
 */
@Serializable
class Coordinates(
    val x: Float, //Поле не может быть null
    val y: Int    //Поле не может быть null
)